docs/contributor/release-process.md

```diff
@@ -1,50 +1,50 @@
-# Backend - Release Workflow Documentation
+# Backend - Release Workflow
 
-This documentation provides a step-by-step guide to the release process workflow for building and pushing a Go project using GitHub Actions. This workflow is triggered on pushes to the `main` branch and tags, specifically for changes within the `assistant/backend` directory.
+This document provides a step-by-step guide to the release process workflow for building and pushing a Go project using GitHub Actions. This workflow is triggered on pushes to the `main` branch and tags, specifically for changes within the `assistant/backend` directory.
 
 ## Prerequisites
 
-- PR to the `main` branch:
-  - label: `lgtm`
-  - Approved by at least one reviewer
-  - Passes all checks (Bckend E2E tests, Linting, Git Leaks)
-
-## Files and folders
+A pull request (PR) to the `main` branch must:
+  * Have the `lgtm` label
+  * Get approval from at least one reviewer <!--code owner?-->
+  * Pass all checks (Bckend E2E tests, Linting, Git Leaks)
 
+## Files and Folders
+<!--what is this list?-->
```

Comment:
I am not sure, and the person who wrote this initially is currently on vacation. My assumption would be, that this is an overview how this repository is structured, but it's outdated. For example `scripts/kubernetes` was renamed to `scripts/k8s`. 

The change is from June. I think this whole document is outdated. We wrote a new job to build the image in July and do not have a release process yet. 

I think this whole file can be deleted/rewritten. I'll create a PR to document the build process.